client/Vitals.ts

```diff
@@ -88,4 +88,36 @@ export class VitalsApi {
       provider
     );
   }
+
+  public async bloodOxygen(
+    userId: string,
+    startDate: Date,
+    endDate?: Date,
+    provider?: string,
+
+  ): Promise<TimeseriesPoint[]> {
+    return this.timeseriesData(
+      userId,
+      'blood_oxygen',
+      startDate,
+      endDate,
+      provider
+    );
+  }
+
+  public async bloodPressure(
+    userId: string,
+    startDate: Date,
+    endDate?: Date,
+    provider?: string,
+
+  ): Promise<TimeseriesPoint[]> {
+    return this.timeseriesData(
+      userId,
+      'blood_pressure',
+      startDate,
+      endDate,
+      provider
+    );
+  }
```

Comment:
I am not sure `blood_pressure` fits on a `TimeseriesPoint`. From [here](https://docs.tryvital.io/api-reference/vitals/blood-pressure):

```
  {
    "id": 2,
    "timestamp": "2022-08-04T12:42:45.727135+00:00",
    "systolic": "125",
    "diastolic": "75",
    "type": "cuff",
    "unit": "mmHg"
  }
```